apps/src/templates/sectionProgress/SectionProgress.jsx

```diff
@@ -82,6 +84,23 @@ class SectionProgress extends Component {
     this.props.loadScript(this.props.scriptId);
   }
 
+  componentDidUpdate(prevProps) {
+    // Check if we are going from a CSF script to a non-CSF script and
+    // currentView is Standards. If so re-set currentView to Summary since
+    // Standards doesn't apply.
+    const isCSFBefore =
+      prevProps.scriptData && !prevProps.scriptData.excludeCsfColumnInLegend;
+    const isCSFAfter =
+      this.props.scriptData && !this.props.scriptData.excludeCsfColumnInLegend;
+    if (
+      prevProps.currentView === ViewType.STANDARDS &&
+      isCSFBefore &&
+      !isCSFAfter
```

Comment:
great catch! I like what you did here, but I think you can make it a bit simpler and not look at the previous state:
```
const isCSF =
      this.props.scriptData && !this.props.scriptData.excludeCsfColumnInLegend;
    if (
      this.props.currentView === ViewType.STANDARDS &&
      !isCSF
```